### PR TITLE
fix(ui/icons): changes in icons svg

### PR DESCRIPTION
**Pull request by BrooklynKing**

### CODE REVIEW

File: src/modules/ui/controls/full-screen/templates/enterFullScreenIcon.dot [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-    <!-- padding-bottom: 100% * height/width -->
-    <path class="${props.themeStyles.fullScreenSvgFill}" fill-rule="evenodd" d="M2 10H0v4h4v-2H2v-2zM0 1v3h2V2h2V0H0v1zm14-1h-4v2h2v2h2V0zm-2 12h-2v2h4v-4h-2v2z"/>
-  </svg>
-</div>
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  viewBox="0 0 26 26"
+  preserveAspectRatio="xMidYMin slice"
+  width="100%">
+  <!-- padding-bottom: 100% * height/width -->
```
Comment (maxvektor) [inline src/modules/ui/controls/full-screen/templates/enterFullScreenIcon.dot]: I think we don't need this anymore.

File: package.json [hunk 2]
```diff
@@ -59,7 +59,8 @@
     "test:unit": "haste-playable test --mocha",
     "pretest:playback": "haste-playable clean && npm run bundle:playback-test && npm run bundle:playback-test-with-adapters",
     "test:playback": "haste-playable test --karma dev-env/karma.conf.js && haste-playable test --karma dev-env/karma-with-adapters.conf.js",
-    "precommit": "lint-staged && npm run test:unit && npm run test:node && npm run documentation:gitadd",
+    "precommit": "lint-staged",
+    "prepush": "npm run test:unit && npm run test:node && npm run documentation:gitadd",
```
Comment (bodia-uz) [inline package.json]: `npm run documentation:gitadd` should be in `precommit` (code diff => doc diff)

Comment (bodia-uz) [reply package.json]: Tests on hooks mb remove at all and allow PR to run them (+disallow to push in master)

Comment (bodia-uz) [reply package.json]: A side effect of "disallow push in master" that we could not push release commits/tags from master

**Pull request merged**